app/assets/stylesheets/_form.scss

```diff
@@ -24,3 +24,14 @@
     }
   }
 }
+
+.date-form-group .auth-form-control {
+  width: auto;
```

Comment:
If you're not following any style guides, order the properties in alphabetical order.